src/proto/grpc/testing/messages.proto

```diff
@@ -38,9 +38,6 @@ package grpc.testing;
 enum PayloadType {
   // Compressable text format.
   COMPRESSABLE = 0;
-
-  // Uncompressable binary format.
-  UNCOMPRESSABLE = 1;
```

Comment:
agreed. deprecating in proto, removing from code (wip, but will be part of this PR)